mmv1/third_party/terraform/services/netapp/resource_netapp_storage_pool_test.go.erb

```diff
@@ -187,6 +187,7 @@ resource "google_netapp_storage_pool" "test_pool" {
   network = data.google_compute_network.default.id
   zone = "us-east1-c"
   replica_zone = "us-east1-b"
+  allow_auto_tiering = true
```

Comment:
Looks like this is not supported, we might need to turn this on in order to continue with testing:
```hcl
        Error: Error waiting to create storagePool: Error waiting for Creating storagePool: Error code 7, message: failed precondition: "Auto Tiering is not supported"
        
          with google_netapp_storage_pool.test_pool,
          on terraform_plugin_test.tf line 3, in resource "google_netapp_storage_pool" "test_pool":
           3: resource "google_netapp_storage_pool" "test_pool" {
```